content/layout/intrinsic-2_nav/index.md

```diff
@@ -0,0 +1,219 @@
+---
+page_type: guide
+title: Nav Layout
+author: adamargyle
+description:
+tags:
+- Layout
+- Design
+- CSS
+web_lighthouse: N/A
+web_updated_on:
+web_published_on:
+wf_blink_components: Blink>Accessibility
+---
```

Comment:
i've added titles to all my documents! i was omitting them because i saw a design comp that looked like it automated the guide title by extracting it from the frontmatter. we'll come to that when we need to! for now i'll put them back in, it's much better with them in there right now.